paasta_tools/marathon_serviceinit.py

```diff
@@ -267,9 +267,6 @@ def status_smartstack_backends(service, instance, job_config, cluster, tasks, ex
     if instance != nerve_ns:
```

Comment:
I would also junk lines 267-269. It isn't N/A anymore if we are outputting stuff